aliyun/src/main/java/org/apache/iceberg/aliyun/oss/OSSOutputFile.java

```diff
@@ -25,15 +25,17 @@
 import org.apache.iceberg.io.InputFile;
 import org.apache.iceberg.io.OutputFile;
 import org.apache.iceberg.io.PositionOutputStream;
+import org.apache.iceberg.metrics.MetricsContext;
 
 class OSSOutputFile extends BaseOSSFile implements OutputFile {
 
-  OSSOutputFile(OSS client, OSSURI uri, AliyunProperties aliyunProperties) {
-    super(client, uri, aliyunProperties);
+  OSSOutputFile(OSS client, OSSURI uri, AliyunProperties aliyunProperties, MetricsContext metrics) {
+    super(client, uri, aliyunProperties, metrics);
   }
 
-  static OSSOutputFile fromLocation(OSS client, String location, AliyunProperties aliyunProperties) {
-    return new OSSOutputFile(client, new OSSURI(location), aliyunProperties);
+  static OSSOutputFile fromLocation(OSS client, String location, AliyunProperties aliyunProperties,
```

Comment:
This is only used in tests as far as I can tell. How about not updating it and passing `nullMetrics` here? That will cut down on unnecessary test changes.

Reply:
Nice catch, I have passed `MetricsContext.nullMetrics()` to reduce the test changes.